vsix/vsix.csproj

```diff
@@ -65,8 +65,9 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Content Include="LICENSE">
+    <Content Include="..\LICENSE">
```

Comment:
this doesn't work (when tested with resulting vsix).  a limitation of vsix creation - the LICENSE file must be copied from parent (where it's shared by Microsoft.Windows.CppWinRT.nuspec) and .gitignored to not be checked in.